aptos-move/framework/aptos-experimental/sources/trading/order_book/bulk_order_book_types.move

```diff
@@ -76,6 +76,7 @@ module aptos_experimental::bulk_order_book_types {
     enum BulkOrderRequest has copy, drop {
         V1 {
             account: address,
+            sequence_number: u64, // sequence number for order validation
```

Comment:
can we be more descriptive on the naming here? Since this is not the account sequence number.

even `order_sequence_number` if there isn't something better.

Reply:
A bit verbose but sure let's change it to order_sequence_number